tests/foreman/virtwho/cli/test_nutanix.py

```diff
@@ -173,3 +173,134 @@ def test_positive_hypervisor_id_option(
+        )
+        hypervisor_name, guest_name = deploy_configure_by_script(
+            script, form_data['hypervisor-type'], debug=True, org=default_org.label
+        )
+        # Check the option "prism_central=true" should be set in etc/virt-who.d/virt-who.conf
+        config_file = get_configure_file(virtwho_config['id'])
+        assert get_configure_option("prism_central", config_file) == 'true'
+        virt_who_instance = target_sat.cli.VirtWhoConfig.info({'id': virtwho_config['id']})[
+            'general-information'
+        ]['status']
+        assert virt_who_instance == 'OK'
+        hosts = [
+            (hypervisor_name, f'product_id={settings.virtwho.sku.vdc_physical} and type=NORMAL'),
+            (guest_name, f'product_id={settings.virtwho.sku.vdc_physical} and type=STACK_DERIVED'),
+        ]
+        for hostname, sku in hosts:
+            host = target_sat.cli.Host.list({'search': hostname})[0]
+            subscriptions = target_sat.cli.Subscription.list(
+                {'organization': default_org.name, 'search': sku}
+            )
+            vdc_id = subscriptions[0]['id']
+            if 'type=STACK_DERIVED' in sku:
+                for item in subscriptions:
+                    if hypervisor_name.lower() in item['type']:
+                        vdc_id = item['id']
+                        break
+            result = target_sat.cli.Host.subscription_attach(
+                {'host-id': host['id'], 'subscription-id': vdc_id}
+            )
+            assert result.strip() == 'Subscription attached to the host successfully.'
+        target_sat.cli.VirtWhoConfig.delete({'name': virtwho_config['name']})
+        assert not target_sat.cli.VirtWhoConfig.exists(search=('name', form_data['name']))
+
+    @pytest.mark.tier2
+    def test_positive_prism_central_prism_flavor_option(
+        self, default_org, form_data, virtwho_config, target_sat
+    ):
+        """Verify prism_flavor option by hammer virt-who-config update"
+
+        :id: 701e0390-2bfb-404a-a9bd-fa0fb5ecfdf8
+
+        :expectedresults: prism_flavor option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        value = 'central'
+        target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'prism-flavor': value})
+        result = target_sat.cli.VirtWhoConfig.info({'id': virtwho_config['id']})
+        assert result['general-information']['ahv-prism-flavor'] == value
+        config_file = get_configure_file(virtwho_config['id'])
+        command = get_configure_command(virtwho_config['id'], default_org.name)
+        deploy_configure_by_command(command, form_data['hypervisor-type'], org=default_org.label)
+        assert get_configure_option("prism_central", config_file) == 'true'
+        target_sat.cli.VirtWhoConfig.delete({'name': virtwho_config['name']})
+        assert not target_sat.cli.VirtWhoConfig.exists(search=('name', form_data['name']))
```

Comment:
Instead of deleting VirtWhoConfig in each test, why not include this step in a teardown of the `virtwho_config` fixture, You can consider this in your optimization PR since it appears to be required in all your tests, feel free to reach out if need help

Reply:
> Instead of deleting VirtWhoConfig in each test, why not include this step in a teardown of the `virtwho_config` fixture, You can consider this in your optimization PR since it appears to be required in all your tests, feel free to reach out if need help
@Gauravtalreja1 Thanks for your suggestion, have already add delete virtwho config and assert part in fixture. Please help to review and merge the PR, thanks.